ci/dhis2-verify-app.yml

```diff
@@ -173,7 +169,7 @@ jobs:
     release:
         runs-on: ubuntu-latest
         needs: [publish]
-        if: "github.event_name == 'push' && !contains(github.event.head_commit.message, '[skip ci]')"
+        if: "!github.event.push.repository.fork && github.actor != 'dependabot[bot]'"
```

Comment:
I figured that we could omit the check for the push event, since in my understanding this workflow is triggered by the push event, and so `github.event_name == 'push'` should always be true. Correct me if I'm wrong.

Reply:
`push` triggers on each `git push`, `pull_request` triggers on updates to a PR.

So if I `git push` to a branch that has a PR, two actions get triggered: `push` and `pull_request`.

If I push to a branch that does not have a PR, only `push` is triggered.

Reply:
Yeah exactly. So my reasoning was that since this workflow will be triggered on push, the check for the push event should be safe to omit.